rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] According to the associations, kindly make sure `Getters` and `Setters` are **present** for `@book` and `@person` instance variables  present i.e `:person` and `:book` should be added to `attr_accessor`